### DISCUSSION TITLE
怎么编译呢？

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#97] [Q&A]
你好，我是一个小白。我下载了武汉大学的模板，并且安装了texworks。但是我看编译时要用到命令行？不能在texworks下编译吗？谢谢

**Comment by earthmoon** [comment]
要是有时间的话，可以线下讨论么，

**Reply by xkwxdyy** [reply]
小白的话建议先读 lshort-zh-cn 了解基础知识再使用

**Reply by earthmoon** [reply]
读过一遍，似懂非懂。
